src/main/webapp/js/go-to-deal.js

```diff
@@ -91,26 +94,67 @@ function loadCommentsToPage(comments) {
 /**
  * Creates comment element
  * @param {object} comment
- * @return {object} commentElement
+ * @return {HTMLDivElement} commentElement
  */
 function createCommentElement(comment) {
   const commentElement = document.createElement('div');
-  commentElement.className = 'border border-info py-3 px-3 my-3';
+  commentElement.className = 'border border-info py-3 px-3 my-3 d-flex';
+
+  const contentElement = document.createElement('div');
+  contentElement.className = `flex-grow-1 d-flex flex-column
```

Comment:
nit: If you use template literals here, it will add a lot of whitespace before `justify-content-between`. Maybe just use a regular string?